test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/BasicSearchTests.cs

```diff
@@ -77,30 +64,15 @@ public async Task GivenResourceWithVariousValues_WhenSearchedWithCityParam_ThenO
 
             // Create various resources.
             Patient[] patients = await Client.CreateResourcesAsync<Patient>(
-                p => SetPatientInfo(p, "seattle", "Robinson"), // match
-                p => SetPatientInfo(p, "Portland", "Williamas"),
-                p => SetPatientInfo(p, "SEATTLE", "Skouras"), // match
-                p => SetPatientInfo(p, "Sea", "Luecke"),
-                p => SetPatientInfo(p, "Seattle", "Jones"), // match
-                p => SetPatientInfo(p, "New York", "Cook"),
-                p => SetPatientInfo(p, "Amsterdam", "Hill"));
+                p => SetPatientInfo(p, "seattle", "Robinson", tag), // match
+                p => SetPatientInfo(p, "Portland", "Williamas", tag),
+                p => SetPatientInfo(p, "SEATTLE", "Skouras", tag), // match
+                p => SetPatientInfo(p, "Sea", "Luecke", tag),
+                p => SetPatientInfo(p, "Seattle", "Jones", tag), // match
+                p => SetPatientInfo(p, "New York", "Cook", tag),
+                p => SetPatientInfo(p, "Amsterdam", "Hill", tag));
 
             await ExecuteAndValidateBundle("Patient?address-city=Seattle", patients[0], patients[2], patients[4]);
```

Comment:
Not part of this PR, but this test defines a tag but doesn't use it in its search.